### ISSUE TITLE
dollar sign disappears when creating new snippet from selection

### ISSUE THREAD

**Issue by sambody** [robole/vscode-snippets-ranger#7]
What happens:
When I select this code in a php file, containing variables that start with dollar,
```php
$statement = $this->pdo->prepare($sql);
```
and then do "Code Ranger : Add new snippet", I get the following result in the snippets json file:

```json
"enter a name" : {
    "prefix": "",
    "body": ["  = ->pdo->prepare( );"],
    "description": ""
},
```

Notice above how all variables that started with a dollar completely disappeared from the selection in the resulting snippet. 

What I expect: 
- The variables should not disappear. 
- The dollar should be included from the original selection, AND escaped like this:
```json
"enter a name" : {
    "prefix": "",
    "body": ["\\$statement = \\$this->pdo->prepare(\\$sql);"],
    "description": ""
},
```

I hope this clarifies the issue. Many thanks.

**Comment by robole** [comment]
hi @sambody ,

this issue should be resolved in v1.0.1. You can confirm it is resolved and close the issue yourself.

**Comment by sambody** [comment]
I confirm it is resolved. Many thanks for that.
